index.html

```diff
@@ -34,4 +34,194 @@ <h1>Kiprono Japhet</h1>
             <img src="images/image_geometry_2.png" alt="image">
         </div>
     </header>
+    <section class="projects">
+        <div class="projects-wrapper">
+            <div class="project-image">
+                <img class="project-img" src="images/Rectangle1.png" alt="image">
+            </div>
+            <div class="project-lg">
+              <div class="project-lg-1">
+                <h3>Project name goes here</h3> 
+                
+                <div class="btn-langs-left">
+                    <button name="btn-langs" type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+                    <button name="btn-langs" type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+                    <button name="btn-langs" type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
```

Comment:
[REQUIRED] Please kindly use `<ul />` for grouping the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") and `<li/>` for each of them as it is required by the project and if we keep buttons here, they can easily affect the `accessibility` of the webpage negatively making confusion for disabled users who use screen readers for navigations: sample code:

```HTML
 <ul>
    <li>Tech-1</li>
    <li>Tech-2</li>
    <li>Tech-3</li>
  </ul>
```
_NOTE: this comment refers to all cards within your `.projects` section please kindly implement request to all of them_

Reply:
ALl good now 👍🏽

Reply:
yes,